gems/smithy/lib/smithy/command.rb

```diff
@@ -27,11 +27,6 @@ def self.gem_options!
         method_option :gem_namespace, type: :string,
                                       desc: 'The namespace of the gem to generate, e.g. `MyGem::Namespace`.' \
                                             'If not provided, the gem name will be used to infer the namespace.'
-        method_option :skip_welds, type: :boolean, default: false,
```

Comment:
Is there not a new skip_rivets?

Reply:
You asked for those to be added in the first place.  I think they're generally useful.  But I'm fine removing it from the CLI options as long as its still supported in the internal plan.

Reply:
I did as a way to skip polishes. But we need welds for like endpoint rules. So the option doesn't make sense anymore with weld and polish as one thing. So now we have to skip based upon `for?`